Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Certificate.java

```diff
@@ -158,11 +164,7 @@ public void setExpiresInDays(long expiresInDays) {
 
     public String getExpiryColorCoding() {
         if (expiryColorCoding == null) {
-            Date certExpiryDate = x509Cert.getNotAfter();
-            Date today = new Date();
-            long dateDiff = certExpiryDate.getTime() - today.getTime();
-            expiresInDays = dateDiff / (24 * 60 * 60 * 1000);
-            if (expiresInDays <= 30) {
+            if (getExpiresInDays() <= 30) {
```

Comment:
Consider to assign getExpiresInDays result into variable so that you don't need to perform calculate over again when calling getExpiresInDays() multiple places

Reply:
The computed value is stored in the variable expiresInDays which is being used further in the logicc.